apps/docs/content/components/forms/HelperMessage.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: HelperMessage
-description: The HelperMessage component is used to display auxiliary text to guide users in the interface.
+description: An HelperMessage component displays auxiliary text to guide users in the interface.
```

Comment:
```suggestion
description: A HelperMessage component displays auxiliary text to guide users in the interface.
```
you changed all the others to not be capitalized so should it be "helper message"?
How do we know when to use what?

Reply:
I changed the ones that could be used in a sentence such as switch, button, etc. helper message felt weird to me, maybe because it has two words?

Reply:
well looking at react spectrum, they don't seem very consistent either... I guess it's fine.